app/src/main/java/net/skyscanner/backpack/demo/compose/ButtonsStory.kt

```diff
@@ -53,95 +61,54 @@ fun ButtonsStory(
   size: BpkButtonSize,
   modifier: Modifier = Modifier,
 ) {
-
-  val icon = when (size) {
-    BpkButtonSize.Default -> BpkIcons.Sm.LongArrowRight
-    BpkButtonSize.Large -> BpkIcons.Lg.LongArrowRight
-  }
-
-  Row(
-    modifier = modifier
-      .fillMaxSize()
-      .padding(horizontal = BpkSpacing.Base)
-      .padding(top = BpkSpacing.Md),
-    verticalAlignment = Alignment.Top,
-    horizontalArrangement = Arrangement.SpaceBetween,
-  ) {
-
-    ButtonsColumn { type, enabled ->
-      BpkButton(
-        text = stringResource(R.string.button),
-        size = size,
-        type = type,
-        loading = loading,
-        enabled = enabled,
-        onClick = ::load,
-      )
-    }
-
-    ButtonsColumn { type, enabled ->
-      BpkButton(
-        text = stringResource(R.string.button),
-        icon = icon,
-        position = BpkButtonIconPosition.Start,
-        size = size,
-        type = type,
-        enabled = enabled,
-        loading = loading,
-        onClick = ::load,
-      )
-    }
-
-    ButtonsColumn { type, enabled ->
-      BpkButton(
-        text = stringResource(R.string.button),
-        icon = icon,
-        position = BpkButtonIconPosition.End,
+  LazyColumn(modifier.fillMaxSize()) {
```

Comment:
separated out the link buttons here and moved back from column based to row based to apply the background for the on dark/light buttons